index.html

```diff
@@ -359,7 +363,7 @@
         var drawing = document.getElementById("drawing");
 
         for (let element of drawing.children) {
-          if (element.tagName === "circle" && element.getAttribute("cx") === x && element.getAttribute("cy") === y && element.getAttribute("r") === "10") {
+          if (element.tagName === "dot" && element.getAttribute("cx") === x && element.getAttribute("cy") === y && element.getAttribute("r") === "10") {
```

Comment:
No. `dot` is not a tag name in SVG. There is no `<dot>`. This needs to still be `circle`.